packages/flutter/lib/src/material/slider.dart

```diff
@@ -717,6 +726,11 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
     // in range_slider.dart.
     Size screenSize() => MediaQuery.of(context).size;
 
+    Map<ShortcutActivator, Intent> _shortcutMap() =>
+        MediaQuery.of(context).navigationMode == NavigationMode.directional
```

Comment:
In this case, It may be more beneficial to use switch case so the lint would warn when if we add a new navigation mode in the future

Reply:
Done.